src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+logging.level.root=INFO
```

Comment:
TBH, it's weird that you need it. Logging always worked for me without any configuration.

Reply:
For the first time I see this as well. I just tried with different maven plugins but couldn't spot the issue. So let's keep this for now.